### ISSUE TITLE
Copied Outlook appointments are not synced to google

### ISSUE THREAD

**Issue by gnubee40** [phw198/OutlookGoogleCalendarSync#326]
Hi!

First of all big thumbs up for this program. I have up to now used commercially software which has ceased working for me. Also, its userinterface is hopelessly cluttered compared to yours. There will be donations!

Still, I get problems. I have 2 different Outlook calendars and 2 different Google calendars i need to be in sync. I sync in parallel, which Means default Outlook syncs with defauls Google, and a another Outlook calendar folder syncs to a different google calendar (same account)

More than that, I copy some events from the subfolder in Outlook calendar over to the default calendar by drag and drop (not possible to get them written in simultaneously.
These copied events dont always get synced to Google by 2-way sync. By reading another thread here, I learned I can workaround by setting Outlook as master and hold Down shift button when pressing sync.

The issue might be, that the copied appointments already were synced in the the subfolder Outlook calendar to the sub-calendar in Google.

Will parallel syncing of 2 different "sub-calendars" on the same Outlook and Google account be possible with this program in the future?

![2017-09-07 16_08_13-outlook google calendar sync](https://user-images.githubusercontent.com/5459195/30167857-fddadf24-93e7-11e7-9dfb-34702a54da0b.png)

**Comment by phw198** [comment]
This sounds like the same underlying problem as issue #224 - the "tags" that OGCS uses inside each appointment also get copied, which then confuses the 2-way sync engine. I need to work out a reliable method of removing these tags upon copy.

**Comment by gnubee40** [comment]
Well - sounds tricky. Read some of the threads you referenced. It is very impressive, that you managed to make such a good solution in your spare time. We can hope a solution emerges at some point. 
All best.

**Comment by phw198** [comment]
Thanks @gnubee40 :slightly_smiling_face: 

Closing this as a duplicate of #224, which I think you've already said is working for you.

**Comment by gnubee40** [comment]
Anytime! Great work!
